### PR TITLE
`GlobalConfig` Rule them all

### PR DESCRIPTION
**Pull request by LemonHX**
added `StoreGlobalConfig` and `LoadGlobalConfig` for pdpd as grpc methods.

### CODE REVIEW

File: proto/pdpb.proto [hunk 1]
```diff
@@ -84,6 +84,39 @@ service PD {
...
+    string name = 1;
+    string value = 2;
+}
+
+message StoreGlobalConfigResponse {
+    Error error = 1;
+}
+
+message LoadGlobalConfigRequest {
+    repeated string name = 1;
```
Comment (crazycs520) [inline proto/pdpb.proto:114]:
```suggestion
    repeated string names = 1;
```

File: proto/pdpb.proto [hunk 2]
```diff
@@ -84,6 +84,39 @@ service PD {
...
+}
+
+message LoadGlobalConfigResponse {
+    repeated string value = 1;
```
Comment (crazycs520) [inline proto/pdpb.proto:118]:
```suggestion
    repeated string values = 1;
```

File: proto/pdpb.proto [hunk 3]
```diff
@@ -84,6 +84,39 @@ service PD {
...
+
+    rpc WatchGlobalConfig(WatchGlobalConfigRequest) returns (stream WatchGlobalConfigResponse) {}
+}
+
+message WatchGlobalConfigRequest {
+
+}
+
+message WatchGlobalConfigResponse {
+    string key = 1;
```
Comment (crazycs520) [inline proto/pdpb.proto:100]:
```suggestion
    string name = 1;
```

File: proto/pdpb.proto [hunk 4]
```diff
@@ -84,6 +84,39 @@ service PD {
...
+    string value = 2;
+}
+
+message StoreGlobalConfigRequest {
+    string name = 1;
+    string value = 2;
+}
+
+message StoreGlobalConfigResponse {
+    Error error = 1;
```
Comment (crazycs520) [inline proto/pdpb.proto:110]: Look like this is field is never used?

Comment (crazycs520) [reply proto/pdpb.proto:110]: Maybe you can just like `LoadGlobalConfigResponse`. just use `ok` filed.

File: proto/pdpb.proto [hunk 5]
```diff
@@ -84,6 +84,39 @@ service PD {
...
+}
+
+message LoadGlobalConfigRequest {
+    repeated string names = 1;
+}
+
+message LoadGlobalConfigResponse {
+    repeated string values = 1;
+    bool ok = 2;
```
Comment (crazycs520) [inline proto/pdpb.proto:119]: Maybe should use `Error` filed, just like `StoreGlobalConfigResponse`.

**Review by JmPotato — APPROVED** [review]

**Review by crazycs520 — APPROVED** [review]

**Pull request merged**